karma.conf.js

```diff
@@ -14,14 +14,14 @@ module.exports = function(config) {
 
 		plugins: [
 			'karma-mocha',
-			'karma-phantomjs-launcher',
+			'karma-phantomjs2-launcher',
 			'karma-webpack'
 		],
 
 
 		// list of files / patterns to load in the browser
 		files: [
-			'http://polyfill.webservices.ft.com/v1/polyfill.js?ua=safari/4',
+			'https://cdn.polyfill.io/v2/polyfill.js',
```

Comment:
After a chat with Andrew about the bug we've been seeing in the Polyfill service, can you add `?flags=gated` to the end of the polyfill URL pls